src/App.js

```diff
@@ -62,33 +63,37 @@ export default function App() {
 
   return (
     <Appcontainer>
-      {isHidden || (
+      {currentPage === 'activity' && (
         <DurationFilter onFilterDurationReset={filterDurationReset} onFilterDurationValue={setDurationFilterValue} />
       )}
-      {isHidden || <TypeFilter onFilterTypeReset={filterTypeReset} onFilterTypeValue={setTypeFiltervalue} />}
-      {isHidden || (
+      {currentPage === 'activity' && (
+        <TypeFilter onFilterTypeReset={filterTypeReset} onFilterTypeValue={setTypeFiltervalue} />
+      )}
+      {currentPage === 'activity' && (
         <ActivityCards
           activities={filteredActivities}
           onSetActivities={setActivities}
-          onSetIsHidden={() => setIsHidden(!isHidden)}
+          onSetIsHidden={() => setCurrentPage('favorites')}
+          onSetIsHidden1={() => setCurrentPage('map')}
```

Comment:
`onSetIsHidden` is not a good function name for what this is. Also, why do you also have `onSetIsHidden1`? I would suggest to have a function `onNavigate` which receives a parameter `page` as a string.
```suggestion
          onNavigate={(page) => setCurrentPage(page)}
```
In your `ActivityCard` you would then pass the parameter when clicking on a button:
```jsx
        <button onClick={() => onNavigate('favorites')}>Favorites</button>
        <button onClick={() => onNavigate('map')}>Map</button>
```
I know you wrote that you will do the navigation and stuff in the next user story, however, I would recommend you to change it.